packages/node_modules/node-red/settings.js

```diff
@@ -85,6 +42,10 @@ module.exports = {
     // lost.
     //credentialSecret: "a-secret-key",
 
+    // To enabled pretty-printing of the flow within the flow file, set the following
```

Comment:
to enable....   and again is this now the default ?

Reply:
It is the default if projects is enabled (because you'll be doing version control etc) - but not otherwise.

Reply:
ok - so just need to be   To enable pretty-printing...

Reply:
I've just changed it to the following:
```
    /** By default, the flow JSON will be formatted over multiple lines making
     * it easier to compare changes when using version control.
     * To disable pretty-printing of the JSON set the following property to false.
     */
    flowFilePretty: true,
```